cougarvision_utils/detect_img.py

```diff
@@ -13,22 +13,25 @@
 from datetime import datetime as dt
 import re
 import sys
+import logging
 import yaml
 from PIL import Image
-from animl import parse_results, classify, split
+from animl import inference, split
 from sageranger import is_target, attach_image, post_event
-from animl.detectMD import detect_MD_batch
+from animl.detect import detect_MD_batch, parse_MD
 
 from cougarvision_utils.cropping import draw_bounding_box_on_image
 from cougarvision_utils.alert import smtp_setup, send_alert
+from cougarvision_visualize.visualize_helper import get_last_file_number
+from cougarvision_visualize.visualize_helper import create_folder
 
 
 with open("config/cameratraps.yml", 'r') as stream:
```

Comment:
This should not be part of the module, this should be run once from the main function